plugins/rest/plugin.spec.yaml

```
@@ -13,6 +13,9 @@ resources:
   source_url: https://github.com/rapid7/insightconnect-plugins/tree/master/plugins/rest
   license_url: https://github.com/rapid7/insightconnect-plugins/blob/master/LICENSE
   docs_url: https://docs.rapid7.com/insightconnect/http-requests
+key_features: ["it can do this", "and that"]
```

feel free to remove this (or change it) then regenerate 😄 
Now would be a good time to make use of this field properly. 
```suggestion
key_features: ["Make RESTful API calls"]
```